mk/lint.mk

```diff
@@ -0,0 +1,15 @@
+.PHONY: install-pre-commit, install-golangci-lint, install-linter, lint, lint-all
```

Comment:
not important comment: I suggest to add the `.PHONY: my-rule` just before the rule it refers as if the rule change the name or it is deleted, can be observed easier and keept in sync easier.